lib/msal-core/samples/react-sample-app/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "./App.css";
+
+import AuthProvider from "./AuthProvider";
+
+class App extends React.Component {
```

Comment:
Probably should define prop types so its clear we are passing these down to the app